### DISCUSSION TITLE
Embedded track: Failed to load resource: the server responded with a status of 401 (Unauthorized)

### DISCUSSION THREAD

**Discussion by lindavanderhoeven** [soundcloud/api#297] [General]
Hi. Track https://soundcloud.com/iamacademy/dag-3-5-daagse-meditatiereeks-helende-meditatie-30/s-MNmhs8zAwkk, when embedded on a website the download button does not work anymore as of juni 7th approximately. I get "Failed to load resource: the server responded with a status of 401 (Unauthorized)" example page: https://iamacademy.nl/meditatie-hm3-3/. Please advice.

**Comment by lindavanderhoeven** [comment]
@soundcloud-ci Is there anyone that can join this discussion please?

**Comment by dpreussler** [comment]
closing as of age, feel free to reopen if needed
